tests/cli/mirroring_smoke_tests/e2e/fluvio-core.bats

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env bats
+
+TEST_HELPER_DIR="$BATS_TEST_DIRNAME/../../test_helper"
+export TEST_HELPER_DIR
+
+load "$TEST_HELPER_DIR"/tools_check.bash
+load "$TEST_HELPER_DIR"/fluvio_dev.bash
+load "$TEST_HELPER_DIR"/bats-support/load.bash
+load "$TEST_HELPER_DIR"/bats-assert/load.bash
+
+setup_file() {
+    CURRENT_DATE=$(date +%Y-%m)
+    export CURRENT_DATE
+
+    REMOTE_NAME=user1
+    export REMOTE_NAME
+    debug_msg "Topic name: $REMOTE_NAME"
+
+    MESSAGE="$(random_string 7)"
+    export MESSAGE
+    debug_msg "$MESSAGE"
+
+    TOPIC_NAME=mirror-topic
+    export TOPIC_NAME
+    debug_msg "Topic name: $TOPIC_NAME"
+
+    REMOTE_PROFILE=local
+    export REMOTE_PROFILE
+    debug_msg "Remote profile: $REMOTE_PROFILE"
+
+    HOME_PROFILE=$($FLUVIO_BIN profile)
+    export HOME_PROFILE
+    debug_msg "Home profile: $HOME_PROFILE"
+}
+
+@test "Can register an remote cluster" {
```

Comment:
instead of just 1 remote, test should cover arbitrary number of remotes.  Probably set to min of 10 remotes.

Reply:
I merged the implementation with two remotes